src/components/DropdownMenu.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <div :class="$style.dropdowns">
+    <div :class="$style.dropdown">
+      <button :class="$style.button" @click="openSort">
+        <div :class="$style.dropdown_trigger">
+          <p :class="$style.title">並び替え</p>
+          <icon name="chevron-down" :class="$style.icon" />
+        </div>
+      </button>
+      <div v-if="state.isOpenSort" :class="$style.dropdown_menu">
+        <p
+          v-for="(sortOrder, index) in sortOrders"
+          :key="index"
+          :class="$style.contents"
+        >
+          {{ sortOrder.str }}
+        </p>
+      </div>
+    </div>
+    <div :class="$style.dropdown">
+      <button :class="$style.button" @click="openOption">
+        <div :class="$style.dropdown_trigger">
+          <p :class="$style.title">フィルター</p>
+          <icon name="chevron-down" :class="$style.icon" />
+        </div>
+      </button>
+      <div v-if="state.isOpenOption" :class="$style.dropdown_menu">
+        <p
+          v-for="(targetedOption, index) in targetedOptions"
+          :key="index"
+          :class="$style.contents"
+        >
+          {{ targetedOption.str }}
+        </p>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { reactive } from 'vue'
+import Icon from '/@/components/UI/Icon.vue'
+
+export default {
+  name: 'DropdownMenu',
+  components: {
+    icon: Icon
```

Comment:
```suggestion
    Icon
```
だけで大丈夫です
上のtemplate内では何も指定しなかったらケバブケースで参照できます(他も同様)